### PR TITLE
Rename clusterrole in 201-channelable-manipulator-clusterrole.yaml

### LINKED ISSUE #406: Fix duplicate "events-channel-addressable-resolver" cluster role
Failed to run the example with this doc https://github.com/google/knative-gcp/blob/master/docs/channel/README.md.In knative-gcp, 201-addressable-resolvers-clusterrole.yaml and 201-channelable-manipulator-clusterrole.yaml defines the same clusterrole events-channel-addressable-resolver. But the label is different. Events-channel-addressable-resolver in 201-addressable-resolvers-clusterrole.yaml gets override. This will cause the source controller fail to list the channels like follows:
`E1108 23:20:12.029867       1 reflector.go:125] knative.dev/pkg/apis/duck/typed.go:60: Failed to list *v1beta1.AddressableType: channels.messaging.cloud.google.com is forbidden: User "system:serviceaccount:knative-eventing:eventing-source-controller" cannot list resource "channels" in API group "messaging.cloud.google.com" at the cluster scope`

### PR DESCRIPTION
**Pull request by capri-xiyue**
Fixes #406 
## Proposed Changes

- Rename clusterrole in 201-channelable-manipulator-clusterrole.yaml

**Release Note**

```release-note
None
```

### CODE REVIEW

File: config/201-channelable-manipulator-clusterrole.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: events-channel-addressable-resolver
+  name: events-channel-manipulator-resolver
```
Comment (grantr) [inline config/201-channelable-manipulator-clusterrole.yaml]: I think this should be `events-channel-channelable-manipulator` to be consistent with https://github.com/knative/eventing/blob/fa35b5fb53a1e4c9b1ae99cda4874cabbebc2f02/config/channels/in-memory-channel/200-channelable-manipulator-clusterrole.yaml#L18 and https://github.com/knative/eventing-contrib/blob/5408c00d064038924c628b784562f52b32dad101/kafka/channel/config/200-channelable-manipulator-clusterrole.yaml#L18

File: config/201-channelable-manipulator-clusterrole.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: events-channel-addressable-resolver
+  name: events-channel-channelable-manipulator-resolver
```
Comment (grantr) [inline config/201-channelable-manipulator-clusterrole.yaml]: Remove the `-resolver` bit

Comment (capri-xiyue) [reply config/201-channelable-manipulator-clusterrole.yaml]: Done

**Pull request merged**